src/containers/GalaxiesSelectorContainer.jsx

```diff
@@ -43,19 +47,47 @@ class GalaxiesSelectorContainer extends React.PureComponent {
       .split('.')[0];
   }
 
+  selectionCallback = d => {
+    const {
+      answers,
+      updateAnswer,
+      activeQuestionId,
+      options: { toggleDataPointsVisibility },
+    } = this.props;
+
+    const qId = toggleDataPointsVisibility || activeQuestionId;
+    const answer = answers[qId];
+    const answerData = !isEmpty(answer) ? uniq([...d, ...answer.data]) : d;
+
+    if (qId) {
+      updateAnswer(qId, answerData);
+    }
+  };
+
   render() {
+    const {
+      answers,
+      options: { toggleDataPointsVisibility },
+    } = this.props;
     const { imagePath, data, name, domain } = this.state;
-    console.log(domain);
+
+    const selectedData = getSelectedGalaxies(
+      answers,
+      toggleDataPointsVisibility
+    );
+
     return (
       <>
         <h2 className="space-bottom">Galaxies Selector</h2>
         <div className="galaxies-selector-images--container">
           <GalaxySelector
             className="galaxies-selector"
+            {...{ selectedData }}
             data={data}
```

Comment:
Why not include `data={data}` in the spead above??? something like `{...{ selectedData, data }}`??

Reply:
true enough: will make this change and amend it to [EPO-2790](https://github.com/lsst-epo/investigation-static/pull/85)